docs/UserGuide.md

```diff
@@ -203,41 +203,37 @@ The company’s information should be listed on the left panel, which includes t
+**Format:** `edit INDEX [c/COMPANY_NAME] [n/RECRUITER_NAME] [r/ROLE] [s/APPLICATION_STATUS] [d/DEADLINE] [e/EMAIL] [p/PHONE_NUMBER] [pr/PRIORITY] [nt/NOTE]`
 
-* Edits the company at the specified INDEX. The index refers to the index number shown in the displayed company list. The index must be a positive integer 1, 2, 3, ...
+* `INDEX`: Edits the company at INDEX number shown in the displayed company list.
+* `[c/COMPANY_NAME]` etc. : type prefix `c/` followed by new company name to edit company name.
 * At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the company will be removed i.e adding of tags is not cumulative.
-* You can remove all the company’s tags by typing t/ without specifying any tags after it.
 
-**Examples:**
-* `edit 2 s/Pending Application r/frontend developer` edits the status and role of the 2nd person to be Pending Application and frontend developer respectively.
-* `edit 3 e/example@abc.com t/` edits the email of the 3rd person to be example@abc.com and clears all existing tags.
+:warning: When editing the note, the existing note of the company will be removed i.e adding of note is not cumulative.
 
-**Acceptable values for each parameter:**
-* INDEX must be a number. If not the entire command will be considered invalid input.
-* INDEX must not be out of bounds. If not the entire command will be considered invalid input.
-* INDEX must be more than zero. If not the entire command will be considered invalid input.
-* No other string separators other than c/, n/, r/, s/, d/, e/, p/. Otherwise, the entire command will be considered invalid and all data inputted will be discarded.
+**Examples:** </br>
+1.Type `edit 2 s/PI r/Frontend Developer` 
+![EditDemo1.png](images/EditDemo1.png)
+2.The 2nd company's the status and role is edited to be Pending Interview and Frontend Developer respectively.
+![EditDemo2.png](images/EditDemo2.png)
 
-**Expected output when command succeeds:**</br>
-`{COMPANY_NAME} company edited.`
 
-**Expected output when command fails:** </br>
-* If INDEX is out of bounds: </br>
-  `The company index provided is invalid`
-* If invalid string separator: </br>
-  `Invalid command format! edit: Edits the details of the company identified by the index number used in the displayed company list. Existing values will be overwritten by the input values.
-  Parameters: INDEX (must be a positive integer) [c/COMPANY_NAME] [n/RECRUITER_NAME] [r/ROLE] [s/APPLICATION_STATUS] [d/DEADLINE] [e/EMAIL] [p/PHONE] [t/TAG]...
-  Example: edit 1 p/91234567 e/johndoe@example.com`
-* If empty input after string separator: </br>
-  `Please enter a valid {field}`
-
-**Expected UI** <br/>
-![EditUI.png](EditUI.png)
+
+### Clear all entries: `clear` ###
+Clears all entries of internship application.
+
+**Format:** `clear`
```

Comment:
Will it be good to include an image here? to show the reader that it is going to be empty.